lib/bindings/http/structured_0_1.js

```diff
@@ -1,7 +1,7 @@
 var axios = require("axios");
 
 function HTTPStructured(configuration){
-  this.config = configuration;
+  this.config = JSON.parse(JSON.stringify(configuration));
```

Comment:
Check out the [line #14](https://github.com/cloudevents/sdk-javascript/blob/77bad8d07f3c707446a5414d0a1e658c2752f8cf/lib/bindings/http/structured_0_1.js#L14). Same suggestion.